response_operations_ui/views/admin.py

```diff
@@ -21,14 +23,21 @@
 @admin_bp.route('/banner', methods=['GET'])
 @login_required
 def banner_admin():
+    breadcrumbs = [{"text": "Banner Admin", "url": ""}]
+    current_banner = admin_controller.current_banner()
     logger.debug("Banner page accessed", user=current_username())
     form = BannerAdminForm(form=request.form)
-
-    current_banner = admin_controller.current_banner()
-    if current_banner:
-        form.banner.data = current_banner
-        logger.debug("Banner set to ", banner=current_banner)
-    return render_template('banner-admin.html', current_banner=current_banner, form=form)
+    dict_of_alerts = get_alert_list()
+    banner_removed = False
+    if 'banner_removed' in session:
```

Comment:
Do we need this variable? The template should be able to just check if the current_banner is set or not, or have a missed something?

Also it seems a little odd to store it in the session, it's not something I'd expect as the banner is global for all users.

Reply:
Looking at the prototype, once the banner is removed the user is redirected back to /admin/banner and a notification is posted confirming banner removal. The session was added to confirm this to that user and then disappear on refresh. https://rops-prototypes.netlify.app/prototypes/alert-banner-management/version-3/04-remove-alert.html

I can change this to just be a persistent banner with either 'No banner currently active' or '{{current_banner}}'?

Reply:
ah ok that makes sense then 👍

Reply:
I just had a thought, we could use flash flash for this notification and we actually do this elsewhere in rops:

https://flask.palletsprojects.com/en/1.1.x/patterns/flashing/

For example:
https://github.com/ONSdigital/response-operations-ui/blob/a0c7f3c34b9013ed6fc84fb11b429f30f572ba63/response_operations_ui/views/reporting_units.py#L176